skill-tree/src/test/java/com/RDS/skilltree/SkillsIntegrationTests.java

```diff
@@ -208,7 +210,8 @@ public void testAPIReturns400_OnNameNotPassedForSkillCreation() {
 
         response.then()
                 .statusCode(400)
-                .body("error", equalTo("Bad Request"));
+                .body("data",equalTo(null))
+                .body("message", equalTo("Name cannot be null "));
```

Comment:
Nit: Why is there a space at the end?

Reply:
Its because if we are having multiple parameters null, then I was trying to append the exception message with space seperation as done here https://github.com/Real-Dev-Squad/skill-tree-backend/pull/75/files#diff-bc0f0ac40c8f847dbe52332b1cd84130b5953839a5baac89855d261cf61f02d9R47